components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -140,6 +140,73 @@ def fileread_gen(fin, fsize, bufsize):
         p += s
     fin.close()
 
+def countAnnotations(annotationlinks):
```

Comment:
I should move these methods somewhere else. Where's a good place for utilility/helper methods @aleksandra-tarkowska ?